sites/authentication/includes/authentication.html

```diff
@@ -0,0 +1,30 @@
+<section  id="wb-so" class="pull-right btn-signin">
```

Comment:
```suggestion
<section  id="wb-so" class="pull-right">
```
I am thinking that we can probably include the pull-right inside the component style. Question: Is the "pull-right" are optional or required in this pattern?

I suggest the following
```suggestion
<section  id="wb-so">
```